### DISCUSSION TITLE
Rfd problems and WebUi hangs on NEW installation

### DISCUSSION THREAD

**Discussion by Jacuo** [OpenCCU/OpenCCU#2258] [🇺🇸 Q&A]
### Describe the issue you are experiencing

I hava installed fresh installtaion of rasberrymatic on my ELV Charly device After few minutes WebUI "freezes" . I think tha it happen mostly when I click "CCU maintenance" button in Control panel. After I can`t use WebUi - it is not responding 
In logs I see error
```
Mar 11 18:00:34 homematic-raspi user.err rfd: XmlRpc transport error calling system.listMethods({"ba14848a-76a1-4852-b5e0-3345148aab80"}) on http://172.20.0.1:9125/RPC2:
Mar 11 18:02:45 homematic-raspi user.err rfd: XmlRpc transport error calling listDevices({"ba14848a-76a1-4852-b5e0-3345148aab80"}) on http://172.20.0.1:9125/RPC2:
Mar 11 18:02:45 homematic-raspi user.debug rfd: PlatformInit()
Mar 11 18:04:56 homematic-raspi user.err rfd: XmlRpc transport error calling system.listMethods({"ba14848a-76a1-4852-b5e0-3345148aab80"}) on http://172.20.0.1:9125/RPC2:

```
I see also warnings WatchDog: rfd-restart
When I manually kiill rfd process WebUi starts responding for some time

What is intersting me address 172.20.0.1 is very strange as it is not address in my network

### Describe the behavior you expected

WebUi works
No error messages in log
No WatchDog: rfd-restart warnings

### Steps to reproduce the issue

1. Install fresh installation of rasberrymatic and configure it
2.In WebUI enter Settings->Control Panel->CCU maintenance
3. WebUi stops responding 
...

### What is the version this bug report is based on?

3.67.10.20230225

### Which base platform are you running?

rpi3 (RaspberryPi3)

### Which HomeMatic/homematicIP radio module are you using?

RPI-RF-MOD

### Anything in the logs that might be useful for us?

```txt
Mar 11 18:00:34 homematic-raspi user.err rfd: XmlRpc transport error calling system.listMethods({"ba14848a-76a1-4852-b5e0-3345148aab80"}) on http://172.20.0.1:9125/RPC2:
Mar 11 18:02:45 homematic-raspi user.err rfd: XmlRpc transport error calling listDevices({"ba14848a-76a1-4852-b5e0-3345148aab80"}) on http://172.20.0.1:9125/RPC2:
Mar 11 18:02:45 homematic-raspi user.debug rfd: PlatformInit()
Mar 11 18:04:56 homematic-raspi user.err rfd: XmlRpc transport error calling system.listMethods({"ba14848a-76a1-4852-b5e0-3345148aab80"}) on http://172.20.0.1:9125/RPC2:
```

**Comment by jens-maus** [comment]
As this can't / isn't a general "issue" or "bug", I converted this into a discussion open for everyone to answer/help.

**Comment by Jacuo** [comment]
Do you have any idea why it calls 172.20.0.1 ?

**Reply by jens-maus** [reply]
Well, only you can know which system in your LAN has the 172.20.0.1.

**Reply by jens-maus** [reply]
Too me this looks like a docker/container IP address commonly used in HomeAssistant, for example. Thus, can be that you simply registered/configured your HomeAssistant incorrectly...

**Comment by Jacuo** [comment]
I don't have Home Assistant. But yes you are right - it is docker address of  zigbee2mqtt container. 
So How it could be possible taht rfd on fresly intalled rasberrymatic tries to connects to zigbee2mqtt ???? Why ?
And why it causes to Webui to hang in suach a way that event after restart I can`t access WebUI ?

**Reply by jens-maus** [reply]
It is the other way around: this docker container has registered itself to the xmlrpc of RaspberryMatic and thus RaspberryMatic tries to connect to it to deliver data to it but obviously can‘t connect to it for some reason. And this actually causes the hang. So make sure this container does not connect/register itself to your RaspberryMatic…

**Reply by Jacuo** [reply]
Looks, that changing firewall rules to strict solved this problem. But for me is still mystery why zigbemqtt could register to Rasberrymatic.
Any way. Isn't it a bug in Rasberrymatic that it hangs becuase of one werong address ? Shouldn`t it be more resilient ?

**Reply by jens-maus** [reply]
Well, `rfd` is a closed source component where only eQ3 has access to. Thus no possibility to investigate or fix anything in it.

And of course you should check how exactly your zigbee2mqtt installation registers itself to RaspberryMatic since this sounds a misconfiguration on your end and could even cause issues in zigbee2mqtt potentially.
